babel.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  presets: [['@vue/app', { useBuiltIns: 'entry' }]],
```

Comment:
Isn't it enough with just transpiling the nullish coalescing operator and the optional chaining? Vue preset will try to do more transformations

```suggestion
```